perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_unixGroupName_namespace.java

```diff
@@ -161,6 +162,14 @@ public void changedAttributeHook(PerunSessionImpl session, Group group, Attribut
 		}
 	}
 
+	@Override
+	public List<String> getDependencies() {
+		List<String> dependencies = new ArrayList<>();
+		dependencies.add(A_G_unixGID_namespace + ":*");
+		dependencies.add(A_R_unixGID_namespace + ":*");
```

Comment:
Should there really be the `//dependencies.add(A_R_unixGroupName_namespace + ":*");` ? I know that this attribute is used in method checkAttributeValue on line 55 but its value is not used.

Reply:
You are right. The best solution for now is to let it there commented, but also add information that we don't want to use this attribute in dependencies because it does not affect value of dependent attribute.